packages/@uppy/companion/src/server/logger.js

```diff
@@ -41,11 +45,12 @@ exports.debug = (msg, tag) => {
  * @param {string | Error} msg the message to log
  * @param {string} tag a unique tag to easily search for this message
  * @param {string} level error | info | debug
+ * @param {function=} color function to display the log in appropriate color
  */
-const log = (msg, tag, level) => {
-  // @TODO add some colors based on log level
+const log = (msg, tag, level, color) => {
   const time = new Date().toISOString()
+  color = color || ((message) => message)
   // exclude msg from template string so values such as error objects
   // can be well formatted
-  console.log(`uppy: ${time} [${level}] ${tag || ''}`, msg)
+  console.log(color(`uppy: ${time} [${level}] ${tag || ''}`), color(msg))
```

Comment:
Should this read `companion: ` actually?

Reply:
yeah that's true, changed it now 👍